Dockerfile

```diff
@@ -29,13 +29,19 @@ RUN apk add --no-cache --initdb --root /out \
     openssh \
     && true
 
+ARG GITHUB_TOKEN
+RUN if [[ -n "${GITHUB_TOKEN}" ]] ; then echo "using GITHUB_TOKEN";  git config --global url."https://${GITHUB_TOKEN}:@github.com/".insteadOf "https://github.com/"; fi
+
 ENV KUBECTL_VERSION v1.11.5
 RUN curl --silent --location "https://dl.k8s.io/${KUBECTL_VERSION}/bin/linux/amd64/kubectl" --output /out/usr/local/bin/kubectl \
     && chmod +x /out/usr/local/bin/kubectl
 
 # Remaining dependencies are controlled by go.mod
 WORKDIR /src
-ENV CGO_ENABLED=0 GOPROXY=https://proxy.golang.org
+ENV CGO_ENABLED=0
+ENV GOPROXY=https://proxy.golang.org
```

Comment:
Now that we are on Go 1.13, we can omit this environment variable but I think we should be explicit. The default value for `GOPROXY` is `https://proxy.golang.org,direct`, so I think we should set it to that.